src/test/java/subway/line/SubwayLineAcceptanceTest.java

```diff
@@ -0,0 +1,184 @@
+package subway.line;
+
+import io.restassured.RestAssured;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.test.annotation.DirtiesContext;
+import subway.helper.SubwayHelper;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.test.annotation.DirtiesContext.ClassMode.BEFORE_EACH_TEST_METHOD;
+
+@DisplayName("지하철 노선 관련 기능")
+@DirtiesContext(classMode = BEFORE_EACH_TEST_METHOD)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
+public class SubwayLineAcceptanceTest {
+
+    private List<String> stationDataSet = Arrays.asList("지하철 역", "새로운 지하철 역", "또 다른 지하철 역"
+            , "또 다른 새로운 지하철 역");
+
+    @DisplayName("4개의 지하철 역을 생성합니다.")
+    @BeforeEach
+    void setUp() {
+        for (String stationName : stationDataSet) {
+            SubwayHelper.지하철_노션에_지하철_역_요청(stationName);
+        }
```

Comment:
```suggestion
        Stream.of("지하철 역", "새로운 지하철 역", "또 다른 지하철 역", "또 다른 새로운 지하철 역")
                        .forEach(SubwayHelper::지하철_노션에_지하철_역_요청);
```

이런식으로 구현할 수도 있을것 같아요 😄

Reply:
피드백 감사합니다!